src/main/java/com/northcoders/makemydayapi/security/SecurityConfiguration.java

```diff
@@ -22,7 +22,12 @@ public AuthenticationManager authenticationManager(AuthenticationConfiguration a
     public SecurityFilterChain filterChain(HttpSecurity http) throws Exception {
         http.csrf(AbstractHttpConfigurer::disable)
                 .authorizeHttpRequests(auth -> auth
-                        .requestMatchers("/api/v1/makemyday/sign-in", "/api/v1/makemyday/sign-up")
+                        .requestMatchers("/api/v1/makemyday/sign-in",
+                                "/api/v1/makemyday/sign-up",
+                                "/api/v1/activities",

```

Comment:
We do not need to include each single endpoint for `permitAll` on them. This as below does exactly same thing, letting all other endpoints publicly available:

```java
.anyRequest()
.permitAll()
```